### PR TITLE
PMM 4515 PMM-4525 Improving Description, Adding support for MongoDB official 

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -47,9 +47,17 @@ pipeline {
...
             description: 'Percona Server for MongoDB version',
             name: 'MO_VERSION')
-        string(
+        text(
             defaultValue: '--addclient=ps,1',
-            description: 'Configure PMM Clients. ps - Percona Server for MySQL, pxc - Percona XtraDB Cluster, --with-proxysql (to be used with pxc only ex. --addclient=pxc,1 --with-proxysql) , ms - MySQL Community Server, md - MariaDB Server, mo - Percona Server for MongoDB, pgsql - Postgre SQL Server',
+            description: '''
+            Configure PMM Clients
+            ms - MySQL (ex. --addclient=ms,1), To Start with Slowlog use (--addclient=ms,1 --query-source=slowlog),
+            ps - Percona Server for MySQL (ex. --addclient=ps,1),
```
Comment (rnovikovP) [inline pmm/aws-staging-start.groovy]: Can we add an example here about Slow log. 
and for MariaDB too  ) 

thanks

Comment (puneet0191) [reply pmm/aws-staging-start.groovy]: I have added for all type, slowlog for Percona server will be added this week, I will fix it then

**Comment by puneet0191** [comment]
it is a lot of options and will make it complicated.

**Comment by puneet0191** [comment]
I have put the query-source as an option, so no need to worry about passing it as parameter, added example and also updated with MongoDB official support

**Pull request merged**